packages/opentelemetry-node/lib/environment.js

```diff
@@ -136,12 +102,56 @@ function restoreEnvironment() {
 }
 
 /**
- * Gets the env var value also checking in the vars pending to be restored
- * @param {string} name
- * @returns {string | undefined}
+ * @typedef {import('@opentelemetry/core').ENVIRONMENT} OtelEnv
+ */
+/**
+ * @typedef {Object} EdotEnv
+ * @property {string[]} OTEL_NODE_RESOURCE_DETECTORS
+ * @property {number} OTEL_METRIC_EXPORT_INTERVAL
+ * @property {number} OTEL_METRIC_EXPORT_TIMEOUT
+ * @property {boolean} ELASTIC_OTEL_METRICS_DISABLED
+ */
+const otelEnv = getEnv();
```

Comment:
FWIW, there was discussion about getting rid of this `getEnv()` facility. Not soon, though.

Reply:
Looking at the criteria of https://github.com/open-telemetry/opentelemetry-js/issues/5172 `getEnv` is used in several packages so I thought it was safe. I'll add a note. Thanks for the heads up :)

Reply:
Yah, should be fine. Just noting. `getEnv()` might survive for a long while.